projects/netgrif-components-core/src/lib/data-fields/user-list-field/models/user-list-field.ts

```diff
@@ -17,4 +18,46 @@ export class UserListField extends DataField<UserListValue> {
         return (!a && !b) ||
             (!!a && !!b && a.userValues.length === b.userValues.length);
     }
+
+    protected calculateValidity(forValidRequired: boolean, formControl: FormControl): boolean {
+        const isDisabled = formControl.disabled;
+        if (forValidRequired) {
+            formControl.enable();
+        }
+        formControl.clearValidators();
+        if (forValidRequired) {
+            formControl.setValidators(this.behavior.required ? [this.requiredTrue] : []);
+        } else {
+            formControl.setValidators(this.resolveFormControlValidators());
```

Comment:
why is this only done in the `else` block? What does `forValidRequired` even do?